edxval/transcript_utils.py

```diff
@@ -162,3 +164,61 @@ def get_api_token(username, api_key):
         api_token = json.loads(response.content.decode('utf-8'))['ApiToken']
 
     return api_token
+
+
+def validate_missing_attributes(provider, attributes, credentials):
+    """
+    Returns error message if provided attributes are not presents in credentials.
+    """
+    error_message = None
+    missing = [attr for attr in attributes if attr not in credentials]
+    if missing:
+        error_message = '{missing} must be specified for {provider}.'.format(
+            provider=provider,
+            missing=' and '.join(missing)
+        )
+
+    return TranscriptionProviderErrorType.MISSING_REQUIRED_ATTRIBUTES, error_message
```

Comment:
Why is it always returning `MISSING_REQUIRED_ATTRIBUTES` error type? Can we return None if we don't set any error message? then we will not have to explicitly set error_type to None here
https://github.com/edx/edx-val/pull/210/commits/085200f496f0c5ac607ffaad379e06844fe33be9#diff-d78075b51c3378064d929ce6515f2181R1248

Reply:
right.